### PR TITLE
49 add static schedules

### PR DESCRIPTION
**Pull request by toastjpg**
Changes

1. The realm database is now loaded from the assets folder of the project with prepopulated static data
   - This was done to avoid a parsing the json data files and inserting on initial app load
2. Adds GTFS static schedule times for bus stops
3. Homepage supplements missing realtime data with scheduled static information
    - GtfsData.combine can be used to combine the static and realtime data maps

### CODE REVIEW

File: app/src/main/java/ca/wheresthebus/data/ModelFactory.kt
Comment (yenuGH) [inline app/src/main/java/ca/wheresthebus/data/ModelFactory.kt:1]: are we not counting arrival times anymore?

Comment (yenuGH) [reply app/src/main/java/ca/wheresthebus/data/ModelFactory.kt:1]: oh i see the json stuff down below nvm

**Review by yenuGH — APPROVED** [review]
i dont know anything about the gtfs stuff but if it doesn't break lgtm 👍

File: app/src/main/java/ca/wheresthebus/data/ModelFactory.kt [hunk 1]
```diff
@@ -107,26 +103,26 @@ class ModelFactory() {
...
-    fun toMongoStopTime(stopTime: StopTime): MongoStopTime {
-        return MongoStopTime().apply {
-            arrivalTime = stopTime.arrivalTime.atZone(ZoneId.systemDefault()).toInstant().toEpochMilli()
-            id = stopTime.id.value
-            routeId = stopTime.routeId.value
-            serviceId = stopTime.serviceId.value
-            tripId = stopTime.tripId.value
-        }
-    }
+//    fun toStopTime(mongoStopTime: MongoStopTime): StopTime {
```
Comment (bryanjhdang) [inline app/src/main/java/ca/wheresthebus/data/ModelFactory.kt:106]: why is this commented out? if we're not using it should be deleted

**Review by bryanjhdang — APPROVED** [review]
Nice work on the static gtfs data. LGTM 👍

**Pull request merged**